automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/apps/AppLifecycleSpec.scala

```diff
@@ -45,22 +45,9 @@ class AppLifecycleSpec
     appType = AppType.Custom
   )
 
-  val cromwellCreateAppRequest = defaultCreateAppRequest.copy(
-    diskConfig = Some(
-      PersistentDiskRequest(
-        randomDiskName(),
-        Some(DiskSize(500)),
-        None,
-        Map.empty
-      )
-    ),
-    appType = AppType.Cromwell
-  )
-
   private val appTestCases = Table(
     ("description", "createAppRequest"),
-    ("create and delete a CUSTOM app", customCreateAppRequest),
-    ("create and delete a CROMWELL app", cromwellCreateAppRequest)
```

Comment:
the test that was initially here moved to AppCreationSpec?

Reply:
Yes. AppLifecycleSpec was only creating an app with disk attached, monitoring it and then deleting it. While AppCreationSpec is additionally re-creating the app and connecting to same disk which was better for Cromwell app's test case. Hence I moved it there.